BetaCameras/OrbbecOpenNI/OrbbecOpenNI.h

```diff
@@ -104,8 +104,8 @@ namespace MetriCam2
 				unsigned int get(void)
 				{
 					//ir_exposure_get in cmd class not yet functional and can destroy the current state of the camera
-					throw gcnew NotImplementedException();
-					//return GetIRExposure();
+					//throw gcnew NotImplementedException();
```

Comment:
See comment, below.

Reply:
That getter should not have changed. I'll reverse that.